src/utils/Draft.js

```diff
@@ -71,66 +78,123 @@ function botRating(botColors, card) {
   return rating;
 }
 
+async function buildDeck(cards, bot) {
+  const nonlands = cards.filter((card) => !card.type_line.toLowerCase().includes('land'));
+  const lands = cards.filter((card) => card.type_line.toLowerCase().includes('land'));
+
+  const sort_fn = function(a, b) {
+    if (bot) {
+      return botRating(bot, b) - botRating(bot, a);
+    } else {
+      return (draft.ratings[b.details.name] = draft.ratings[a.details.name]);
+    }
+  };
+
+  nonlands.sort(sort_fn);
+  lands.sort(sort_fn);
+
+  const main = nonlands.slice(0, 23).concat(lands.slice(0, 17));
+  const side = nonlands.slice(23).concat(lands.slice(17));
+
+  const deck = [];
+  const sideboard = [];
+  for (let i = 0; i < 16; i += 1) {
+    deck.push([]);
+    if (i < 8) {
+      sideboard.push([]);
+    }
+  }
+
+  for (const card of main) {
+    let index = Math.min(card.cmc, 7);
+    if (!card.type_line.toLowerCase().includes('creature')) {
+      index += 8;
+    }
+    deck[index].push(card);
+  }
+  for (const card of side) {
+    sideboard[Math.min(card.cmc, 7)].push(card);
+  }
+
+  return {
+    deck,
+    sideboard,
+  };
+}
+
 function botPicks() {
   // make bots take one pick out of active packs
-  for (let botIndex = 1; botIndex < draft.packs.length; botIndex++) {
-    const botPack = draft.packs[botIndex][0];
-    const botColors = draft.bots[botIndex - 1];
+  for (let botIndex = 1; botIndex < draft.seats.length; botIndex++) {
```

Comment:
If not subtracting 1 from index inside the loop, do you still want to start from 1?

Reply:
0 is the player seat, bot seats start at 1